src/app/header/header.component.html

```diff
@@ -0,0 +1,32 @@
+<header id="stlHeaderOuter">
+  <!-- insert text message component here -->
```

Comment:
Add a TODO into this comment so it jumps out that this is something that needs to be done. Otherwise this kind of comment can easily be overlooked.